firmware/qemu/src/bin/dbg.rs

```diff
@@ -20,9 +21,11 @@ fn main() -> ! {
     let _: (i32, i32) = dbg!(x - 2, x + 2);
 
     // dbg! with zero arguments
+    #[allow(clippy::let_unit_value)]
     let _: () = dbg!();
```

Comment:
This line could instead be `let () = dbg!();`, which is a bit cleaner and might also silence the warning

Reply:
Kk. But it sadly does not silence the warning.